ansible/roles/volume/tasks/main.yml

```diff
@@ -1,39 +1,49 @@
-- name: check if the external volume already has a filesystem
-  command: "lsblk --fs --json {{ external_volume }}"
+---
+
+- name: Check if the external volume already has a filesystem
+  ansible.builtin.command: "lsblk --fs --json {{ volume_device }}"
   register: lsblk
+  changed_when: false
```

Comment:
@malor This is nice changed. With `changed_when: false`, the task is assumed to be read-only and hence won't be detected as `changed` during a playbook execution.